modules/developers-guide/pages/cql-using.adoc

```diff
@@ -33,37 +33,49 @@ include::quickstart:page$quick_start-rest.adoc[tag=getDockerImage]
 
 include::quickstart:page$quick_start-rest.adoc[tag=startDocker]
 
-=== Installing standalone cqlsh using Docker
+== Using the CQL API using cqlsh
+
+=== Using the Stargate docker image cqlsh
+
+Run `cqlsh` from the Docker image.
```

Comment:
This should be

1. Pull the cqlsh docker image
```
docker pull nuvo/docker-cqlsh
```

2. Grab the IP of your Stargate container (note this assumes that the Stargate container has the name "stargate")
```
docker inspect -f '{{range .NetworkSettings.Networks}}{{.IPAddress}}{{end}}' stargate
```

3. Connect to your Stargate instance using cqlsh
```
docker run --rm -it nuvo/docker-cqlsh cqlsh {ip-address} --cqlversion=3.4.5
```

Reply:
understand now!